--- expected
+++ actual
@@ -12,7 +12,7 @@
 /* harmony import */var _test__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./test */"./test.js");
 
 function myanswer() {
-    _test__WEBPACK_IMPORTED_MODULE_0__["default"];
+    _test__WEBPACK_IMPORTED_MODULE_0__.Z;
 }
 var __WEBPACK_DEFAULT_EXPORT__ = myanswer;
 }),
@@ -20,7 +20,7 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  "default": function() { return __WEBPACK_DEFAULT_EXPORT__; }
+  Z: function() { return __WEBPACK_DEFAULT_EXPORT__; }
 });
 function test() {}
 var __WEBPACK_DEFAULT_EXPORT__ = test;
@@ -35,7 +35,7 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  answer: function() { return answer; }
+  K: function() { return answer; }
 });
  const answer = 30;
 }),
@@ -45,7 +45,7 @@
 /* harmony import */var _app__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./app */"./app.js");
 
 const a = test(()=>__webpack_require__.el("./index.js@51:66").then(__webpack_require__.bind(__webpack_require__, /*! ./lib */"./lib.js")));
-(0, _app__WEBPACK_IMPORTED_MODULE_0__.answer)();
+(0, _app__WEBPACK_IMPORTED_MODULE_0__.K)();
 a;
 }),
 
