--- expected
+++ actual
@@ -11,14 +11,14 @@
 });
 /* harmony import */var _lib__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./lib */"./lib.js");
 
- class Lib extends /* "./lib" unused */null {
+ class Lib extends (/* unused pure expression or super */ null && (OriginLib)) {
 }
 function foo() {
     return {
-        OriginLib: /* "./lib" unused */null
+        OriginLib
     };
 }
- const v = _lib__WEBPACK_IMPORTED_MODULE_0__.value;
+ const v = _lib__WEBPACK_IMPORTED_MODULE_0__.S;
 }),
 "./index.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
@@ -31,7 +31,7 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  value: function() { return value; }
+  S: function() { return value; }
 });
  class Lib {
 }
