--- expected
+++ actual
@@ -6,17 +6,11 @@
 "./child/child/index.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  'value': function() { return value; }
-});
  const value = "dynamic";
 },
 "./child/index.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  'value': function() { return value; }
-});
  const value = "dynamic";
 },
 "./child Sync  recursive ^\\.\\/.*\\.js$": function (module, exports, __webpack_require__) {
