--- expected
+++ actual
@@ -7,9 +7,9 @@
 'use strict';
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  'something': function() { return _lib__WEBPACK_IMPORTED_MODULE_0_["default"]; }
+  'something': function() { return _lib__WEBPACK_IMPORTED_MODULE_0_["default"]; },
+  'b': function() { return _src_a__WEBPACK_IMPORTED_MODULE_1_["default"]; }
 });
-/* harmony import */var _lib__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./lib */"./lib.js");
 
 
 },
@@ -24,13 +24,20 @@
 'use strict';
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  'default': function() { return __WEBPACK_DEFAULT_EXPORT__; }
+  'something': function() { return something; }
 });
  const secret = "888";
  const result = 20000;
  const something = function() {};
 function __WEBPACK_DEFAULT_EXPORT__(){}
 },
+"./src/a.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
+'use strict';
+__webpack_require__.r(__webpack_exports__);
+var __WEBPACK_DEFAULT_EXPORT__ = (()=>{
+    console.log("");
+});
+},
 
 },function(__webpack_require__) {
 var __webpack_exec__ = function(moduleId) { return __webpack_require__(__webpack_require__.s = moduleId) }
