--- expected
+++ actual
@@ -17,16 +17,13 @@
 __webpack_require__.r(__webpack_exports__);
 /* harmony import */var _lib__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/* ./lib */"./lib.js");
 
-_lib__WEBPACK_IMPORTED_MODULE_0__.Lib.a;
+_lib__WEBPACK_IMPORTED_MODULE_0__.a;
 }),
 "./lib.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
 __webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  Lib: function() { return _answer__WEBPACK_IMPORTED_MODULE_0__; }
-});
 /* harmony import */var _answer__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/* ./answer */"./answer.js");
-
+__webpack_require__.es(_answer__WEBPACK_IMPORTED_MODULE_0__, __webpack_exports__);
 
 }),
 
