--- expected
+++ actual
@@ -15,7 +15,7 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  Select: function() { return _bar__WEBPACK_IMPORTED_MODULE_0__["default"]; }
+  Select: function() { return /* reexport safe */ _bar__WEBPACK_IMPORTED_MODULE_0__["default"]; }
 });
 /* harmony import */var _bar__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./bar */"./bar.js");
 
