--- expected
+++ actual
@@ -21,12 +21,12 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  a: function() { return a; }
+  a: function() { return a; },
+  b: function() { return /* reexport safe */ _foo__WEBPACK_IMPORTED_MODULE_0__.b; },
+  c: function() { return /* reexport safe */ _foo__WEBPACK_IMPORTED_MODULE_0__.c; }
 });
 /* harmony import */var _foo__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./foo */"./foo.js");
-__webpack_require__.es(_foo__WEBPACK_IMPORTED_MODULE_0__, __webpack_exports__);
 /* harmony import */var _bar__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/*! ./bar */"./bar.js");
-__webpack_require__.es(_bar__WEBPACK_IMPORTED_MODULE_1__, __webpack_exports__);
 
 
 const a = 3;
@@ -35,10 +35,10 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  b: function() { return b; }
+  b: function() { return b; },
+  c: function() { return /* reexport safe */ _a_js__WEBPACK_IMPORTED_MODULE_0__.c; }
 });
 /* harmony import */var _a_js__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./a.js */"./a.js");
-__webpack_require__.es(_a_js__WEBPACK_IMPORTED_MODULE_0__, __webpack_exports__);
 
 const a = 'foo';
 const b = 'foo';
