--- expected
+++ actual
@@ -6,13 +6,10 @@
 "./a.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
 __webpack_require__.r(__webpack_exports__);
+__webpack_require__.d(__webpack_exports__, {
+  c: function() { return c; }
+});
  const c = 'a';
-}),
-"./bar.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
-"use strict";
-__webpack_require__.r(__webpack_exports__);
- const a = 'bar';
- const c = 'bar';
 }),
 "./c.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
@@ -20,6 +17,8 @@
 __webpack_require__.d(__webpack_exports__, {
   a: function() { return a; }
 });
+/* harmony import */var _foo__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/* ./foo */"./foo.js");
+__webpack_require__.es(_foo__WEBPACK_IMPORTED_MODULE_0__, __webpack_exports__);
 
 
  const a = 3;
@@ -27,6 +26,11 @@
 "./foo.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
 __webpack_require__.r(__webpack_exports__);
+__webpack_require__.d(__webpack_exports__, {
+  b: function() { return b; }
+});
+/* harmony import */var _a_js__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/* ./a.js */"./a.js");
+__webpack_require__.es(_a_js__WEBPACK_IMPORTED_MODULE_0__, __webpack_exports__);
 
  const a = 'foo';
  const b = 'foo';
