diff --git a/dist/api/build.js b/dist/api/build.js
index dbee3d0297107b6a1373575b9caa5f32b5e463bb..c76b32af87e57903523aec08a8ccb1a1b1fe204d 100644
--- a/dist/api/build.js
+++ b/dist/api/build.js
@@ -347,6 +347,8 @@ class Builder {
             });
         }
         catch (e) {
+            console.error("//////////////// Failed to copy artifact ////////////")
+            console.error(e);
             throw new Error('Failed to copy artifact', {
                 cause: e,
             });