diff --git a/types.d.ts b/types.d.ts
index cf0ea711c417bb41cadb1b75957433456c8e7958..3fe3398978312771832971b45e0bf41541614153 100644
--- a/types.d.ts
+++ b/types.d.ts
@@ -407,7 +407,7 @@ declare interface StreamChunksOptions {
 	finalSource?: boolean;
 	columns?: boolean;
 }
-declare namespace exports {
+
 	export namespace util {
 		export namespace stringBufferUtils {
 			export let disableDualStringBufferCaching: () => void;
@@ -456,6 +456,6 @@ declare namespace exports {
 		GeneratedSourceInfo,
 		StreamChunksOptions,
 	};
-}
 
-export = exports;
+
+
