--- expected
+++ actual
@@ -3,9 +3,8 @@
 ---
 ```js title=main.js
 (self['webpackChunkwebpack'] = self['webpackChunkwebpack'] || []).push([["main"], {
-"./index.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
+"./index.js": (function (__unused_webpack_module, __unused_webpack___webpack_exports__, __webpack_require__) {
 "use strict";
-__webpack_require__.r(__webpack_exports__);
 /* harmony import */var _stuff__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./stuff */"./stuff.js");
 
 (0, _stuff__WEBPACK_IMPORTED_MODULE_0__.bar)();
@@ -24,7 +23,6 @@
 }),
 "./stuff.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
-__webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
   bar: function() { return bar; },
   baz: function() { return Baz; }
