--- expected
+++ actual
@@ -3,12 +3,22 @@
 ---
 ```js title=main.js
 (self['webpackChunkwebpack'] = self['webpackChunkwebpack'] || []).push([["main"], {
+"../node_modules/pmodule/a.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
+'use strict';
+__webpack_require__.r(__webpack_exports__);
+/* harmony import */var _tracker__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./tracker */"../node_modules/pmodule/tracker.js");
+var a = "a";
+var b = "b";
+var c = "c";
+
+
+(0, _tracker__WEBPACK_IMPORTED_MODULE_0_.track)("a.js");
+},
 "../node_modules/pmodule/b.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  'z': function() { return _c__WEBPACK_IMPORTED_MODULE_0_.z; },
-  'x': function() { return x; }
+  'z': function() { return _c__WEBPACK_IMPORTED_MODULE_0_.z; }
 });
 /* harmony import */var _c__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./c */"../node_modules/pmodule/c.js");
 /* harmony import */var _tracker__WEBPACK_IMPORTED_MODULE_1_ = __webpack_require__(/* ./tracker */"../node_modules/pmodule/tracker.js");
@@ -22,9 +32,6 @@
 "../node_modules/pmodule/c.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  'z': function() { return z; }
-});
 /* harmony import */var _tracker__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./tracker */"../node_modules/pmodule/tracker.js");
 var z = "z";
 
@@ -36,9 +43,12 @@
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
   'x': function() { return _b__WEBPACK_IMPORTED_MODULE_1_.x; },
+  'y': function() { return _b__WEBPACK_IMPORTED_MODULE_1_.y; },
   'z': function() { return _b__WEBPACK_IMPORTED_MODULE_1_.z; },
   'default': function() { return __WEBPACK_DEFAULT_EXPORT__; }
 });
+/* harmony import */var _a__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./a */"../node_modules/pmodule/a.js");
+__webpack_require__.es(_a__WEBPACK_IMPORTED_MODULE_0_, __webpack_exports__);
 /* harmony import */var _b__WEBPACK_IMPORTED_MODULE_1_ = __webpack_require__(/* ./b */"../node_modules/pmodule/b.js");
 /* harmony import */var _tracker__WEBPACK_IMPORTED_MODULE_2_ = __webpack_require__(/* ./tracker */"../node_modules/pmodule/tracker.js");
 
