--- expected
+++ actual
@@ -7,10 +7,8 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  z: function() { return _c__WEBPACK_IMPORTED_MODULE_0__.z; },
   x: function() { return x; }
 });
-/* harmony import */var _c__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/* ./c */"../node_modules/pmodule/c.js");
 /* harmony import */var _tracker__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/* ./tracker */"../node_modules/pmodule/tracker.js");
 var x = "x";
 var y = "y";
@@ -35,11 +33,8 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  x: function() { return _b__WEBPACK_IMPORTED_MODULE_1__.x; },
-  z: function() { return _b__WEBPACK_IMPORTED_MODULE_1__.z; },
   "default": function() { return __WEBPACK_DEFAULT_EXPORT__; }
 });
-/* harmony import */var _b__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/* ./b */"../node_modules/pmodule/b.js");
 /* harmony import */var _tracker__WEBPACK_IMPORTED_MODULE_2__ = __webpack_require__(/* ./tracker */"../node_modules/pmodule/tracker.js");
 
 
@@ -67,7 +62,7 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 /* harmony import */var pmodule_tracker__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/* pmodule/tracker */"../node_modules/pmodule/tracker.js");
-/* harmony import */var pmodule__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/* pmodule */"../node_modules/pmodule/index.js");
+/* harmony import */var pmodule__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/* pmodule */"../node_modules/pmodule/b.js");
 
 
 
