--- expected
+++ actual
@@ -31,7 +31,6 @@
 __webpack_require__.d(__webpack_exports__, {
   mod: function() { return mod; }
 });
-/* harmony import */var _package1_script__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/* ./package1/script */"./package1/script.js");
 /* harmony import */var _package2_script__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/* ./package2/script */"./package2/script.js");
 
 
@@ -43,20 +42,10 @@
 __webpack_require__.d(__webpack_exports__, {
   exportDefaultUsed: function() { return exportDefaultUsed; }
 });
-/* harmony import */var _script1__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/* ./script1 */"./package1/script1.js");
-__webpack_require__.es(_script1__WEBPACK_IMPORTED_MODULE_0__, __webpack_exports__);
 
-var __WEBPACK_DEFAULT_EXPORT__ = /* "./script1" unused */null;
+var __WEBPACK_DEFAULT_EXPORT__ = (/* unused pure expression or super */ null && (mod));
 
  const exportDefaultUsed = false;
-}),
-"./package1/script1.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
-"use strict";
-__webpack_require__.r(__webpack_exports__);
-/* harmony import */var _script2__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/* ./script2 */"./package1/script2.js");
-__webpack_require__.es(_script2__WEBPACK_IMPORTED_MODULE_0__, __webpack_exports__);
-
-var __WEBPACK_DEFAULT_EXPORT__ = 1;
<<<<<<< HEAD
<<<<<<< HEAD
 },
 "./package1/script2.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
@@ -66,7 +55,7 @@
 });
 
 function __WEBPACK_DEFAULT_EXPORT__(){
-    return /* "./script3" unused */null;
+    return mod;
 }
 
  const exportDefaultUsed = false;
@@ -79,7 +68,6 @@
=======
-})
-,
=======
 }),
>>>>>>> 49860ef57 (feat(runtime-diff): fix tc)
 "./package1/script2.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
@@ -64,13 +53,22 @@
 __webpack_require__.d(__webpack_exports__, {
   exportDefaultUsed: function() { return exportDefaultUsed; }
 });
+/* harmony import */var _script3__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/* ./script3 */"./package1/script3.js");
 
 function __WEBPACK_DEFAULT_EXPORT__(){
-    return /* "./script3" unused */null;
+    return _script3__WEBPACK_IMPORTED_MODULE_0__["default"];
 }
 
  const exportDefaultUsed = false;
 }),
+"./package1/script3.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
+"use strict";
+__webpack_require__.r(__webpack_exports__);
+__webpack_require__.d(__webpack_exports__, {
+  "default": function() { return __WEBPACK_DEFAULT_EXPORT__; }
+});
+var __WEBPACK_DEFAULT_EXPORT__ = 1;
+}),
 "./package2/script.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
 __webpack_require__.r(__webpack_exports__);
<<<<<<< HEAD
@@ -84,11 +82,10 @@
>>>>>>> 674fd416a (feat(runtime-diff): fix rust tc)
=======
@@ -79,11 +77,10 @@
>>>>>>> 49860ef57 (feat(runtime-diff): fix tc)
   exportDefaultUsed: function() { return exportDefaultUsed; }
 });
 /* harmony import */var _script1__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/* ./script1 */"./package2/script1.js");
-__webpack_require__.es(_script1__WEBPACK_IMPORTED_MODULE_0__, __webpack_exports__);
 
 var __WEBPACK_DEFAULT_EXPORT__ = _script1__WEBPACK_IMPORTED_MODULE_0__["default"];
 
<<<<<<< HEAD
=======
- const exportDefaultUsed = true;
+ const exportDefaultUsed = false;
 }),
 "./package2/script1.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
<<<<<<< HEAD
>>>>>>> 674fd416a (feat(runtime-diff): fix rust tc)
=======
 "use strict";
>>>>>>> 49860ef57 (feat(runtime-diff): fix tc)
