--- expected
+++ actual
@@ -43,18 +43,11 @@
 __webpack_require__.d(__webpack_exports__, {
   exportDefaultUsed: function() { return exportDefaultUsed; }
 });
-/* harmony import */var _script1__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/* ./script1 */"./package1/script1.js");
 
-var __WEBPACK_DEFAULT_EXPORT__ = /* "./script1" unused */null;
+var __WEBPACK_DEFAULT_EXPORT__ = (/* unused pure expression or super */ null && (mod));
 
  const exportDefaultUsed = false;
 }),
-"./package1/script1.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
-"use strict";
-__webpack_require__.r(__webpack_exports__);
-
-var __WEBPACK_DEFAULT_EXPORT__ = 1;
-}),
 "./package1/script2.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
 __webpack_require__.r(__webpack_exports__);
@@ -63,7 +56,7 @@
 });
 
 function __WEBPACK_DEFAULT_EXPORT__(){
-    return /* "./script3" unused */null;
+    return mod;
 }
 
  const exportDefaultUsed = false;
