--- expected
+++ actual
@@ -46,7 +46,7 @@
 /* harmony import */var _script1__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./script1 */"./package1/script1.js");
 __webpack_require__.es(_script1__WEBPACK_IMPORTED_MODULE_0__, __webpack_exports__);
 
-var __WEBPACK_DEFAULT_EXPORT__ = /* "./script1" unused */null;
+var __WEBPACK_DEFAULT_EXPORT__ = (/* unused pure expression or super */ null && (mod));
 
  const exportDefaultUsed = false;
 }),
@@ -66,7 +66,7 @@
 });
 
 function __WEBPACK_DEFAULT_EXPORT__(){
-    return /* "./script3" unused */null;
+    return mod;
 }
 
  const exportDefaultUsed = false;
@@ -79,7 +79,6 @@
   exportDefaultUsed: function() { return exportDefaultUsed; }
 });
 /* harmony import */var _script1__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./script1 */"./package2/script1.js");
-__webpack_require__.es(_script1__WEBPACK_IMPORTED_MODULE_0__, __webpack_exports__);
 
 var __WEBPACK_DEFAULT_EXPORT__ = _script1__WEBPACK_IMPORTED_MODULE_0__["default"];
 
