--- expected
+++ actual
@@ -23,7 +23,7 @@
 /* harmony import */var _module__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./module */"./module.js");
 
  function test() {
-    (0, _module__WEBPACK_IMPORTED_MODULE_0__["default"])({
+    (0, _module__WEBPACK_IMPORTED_MODULE_0__.Z)({
         type: "inline"
     });
 }
@@ -41,7 +41,7 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  "default": function() { return __WEBPACK_DEFAULT_EXPORT__; }
+  Z: function() { return __WEBPACK_DEFAULT_EXPORT__; }
 });
 /* harmony import */var _some_module__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./some-module */"./some-module.js");
 
