--- expected
+++ actual
@@ -16,7 +16,6 @@
 (self['webpackChunkwebpack'] = self['webpackChunkwebpack'] || []).push([["main"], {
 "./import-module.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
-__webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
   test: function() { return test; }
 });
@@ -39,7 +38,6 @@
 }),
 "./module.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
-__webpack_require__.r(__webpack_exports__);
 /* harmony import */var _some_module__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./some-module */"./some-module.js");
 
 function getType(obj) {
