--- expected
+++ actual
@@ -8,13 +8,13 @@
 __webpack_require__.r(__webpack_exports__);
 /* harmony import */var _test__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./test */"./test.js");
 
-_test__WEBPACK_IMPORTED_MODULE_0__.obj.test = 1;
+_test__WEBPACK_IMPORTED_MODULE_0__.j.test = 1;
 }),
 "./test.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  obj: function() { return obj; }
+  j: function() { return obj; }
 });
  const obj = {};
 }),
