--- expected
+++ actual
@@ -6,9 +6,6 @@
 "./a.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  'a': function() { return a; }
-});
  const a = {
     a: ''
 };
@@ -16,9 +13,6 @@
 "./b.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  'b': function() { return b; }
-});
  const b = {
     b: ""
 };
