--- expected
+++ actual
@@ -10,9 +10,7 @@
   b: function() { return b; }
 });
 /* harmony import */var _foo_js__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./foo.js */"./foo.js");
-__webpack_require__.es(_foo_js__WEBPACK_IMPORTED_MODULE_0__, __webpack_exports__);
 /* harmony import */var _result_js__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/*! ./result.js */"./result.js");
-__webpack_require__.es(_result_js__WEBPACK_IMPORTED_MODULE_1__, __webpack_exports__);
 function b() {}
 
 
@@ -21,9 +19,7 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 /* harmony import */var _bar_js__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./bar.js */"./bar.js");
-__webpack_require__.es(_bar_js__WEBPACK_IMPORTED_MODULE_0__, __webpack_exports__);
 /* harmony import */var _result_js__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/*! ./result.js */"./result.js");
-__webpack_require__.es(_result_js__WEBPACK_IMPORTED_MODULE_1__, __webpack_exports__);
 const a = 3;
 const b = 3;
 
@@ -40,9 +36,7 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 /* harmony import */var _foo_js__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./foo.js */"./foo.js");
-__webpack_require__.es(_foo_js__WEBPACK_IMPORTED_MODULE_0__, __webpack_exports__);
 /* harmony import */var _bar_js__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/*! ./bar.js */"./bar.js");
-__webpack_require__.es(_bar_js__WEBPACK_IMPORTED_MODULE_1__, __webpack_exports__);
 const c = 103330;
 const b = 103330;
 
