--- expected
+++ actual
@@ -9,23 +9,11 @@
 __webpack_require__.d(__webpack_exports__, {
   'b': function() { return b; }
 });
-/* harmony import */var _foo_js__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./foo.js */"./foo.js");
+/* harmony import */var _foo_js__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(Object(function webpackMissingModule() { var e = new Error('Cannot find module './foo.js''); e.code = 'MODULE_NOT_FOUND'; throw e; }()));
 __webpack_require__.es(_foo_js__WEBPACK_IMPORTED_MODULE_0_, __webpack_exports__);
-/* harmony import */var _result_js__WEBPACK_IMPORTED_MODULE_1_ = __webpack_require__(/* ./result.js */"./result.js");
+/* harmony import */var _result_js__WEBPACK_IMPORTED_MODULE_1_ = __webpack_require__(Object(function webpackMissingModule() { var e = new Error('Cannot find module './result.js''); e.code = 'MODULE_NOT_FOUND'; throw e; }()));
 __webpack_require__.es(_result_js__WEBPACK_IMPORTED_MODULE_1_, __webpack_exports__);
  function b() {}
-
-
-},
-"./foo.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
-'use strict';
-__webpack_require__.r(__webpack_exports__);
-/* harmony import */var _bar_js__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./bar.js */"./bar.js");
-__webpack_require__.es(_bar_js__WEBPACK_IMPORTED_MODULE_0_, __webpack_exports__);
-/* harmony import */var _result_js__WEBPACK_IMPORTED_MODULE_1_ = __webpack_require__(/* ./result.js */"./result.js");
-__webpack_require__.es(_result_js__WEBPACK_IMPORTED_MODULE_1_, __webpack_exports__);
- const a = 3;
- const b = 3;
 
 
 },
@@ -35,18 +23,6 @@
 /* harmony import */var _bar_js__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./bar.js */"./bar.js");
 
 (0, _bar_js__WEBPACK_IMPORTED_MODULE_0_.b)();
-},
-"./result.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
-'use strict';
-__webpack_require__.r(__webpack_exports__);
-/* harmony import */var _foo_js__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./foo.js */"./foo.js");
-__webpack_require__.es(_foo_js__WEBPACK_IMPORTED_MODULE_0_, __webpack_exports__);
-/* harmony import */var _bar_js__WEBPACK_IMPORTED_MODULE_1_ = __webpack_require__(/* ./bar.js */"./bar.js");
-__webpack_require__.es(_bar_js__WEBPACK_IMPORTED_MODULE_1_, __webpack_exports__);
- const c = 103330;
- const b = 103330;
-
-
 },
 
 },function(__webpack_require__) {
