--- expected
+++ actual
@@ -13,28 +13,12 @@
 
 
 }),
-"./foo.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
-"use strict";
-__webpack_require__.r(__webpack_exports__);
- const a = 3;
- const b = 3;
-
-
-}),
 "./index.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 /* harmony import */var _bar_js__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/* ./bar.js */"./bar.js");
 
 (0, _bar_js__WEBPACK_IMPORTED_MODULE_0__.b)();
-}),
-"./result.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
-"use strict";
-__webpack_require__.r(__webpack_exports__);
- const c = 103330;
- const b = 103330;
-
-
 }),
 
 },function(__webpack_require__) {
