--- expected
+++ actual
@@ -7,7 +7,7 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  q: function() { return _lib__WEBPACK_IMPORTED_MODULE_0__.question; }
+  q: function() { return _lib__WEBPACK_IMPORTED_MODULE_0__.w; }
 });
 /* harmony import */var _lib__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./lib */"./lib.js");
 
@@ -24,7 +24,7 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  question: function() { return question; }
+  w: function() { return question; }
 });
  const answer = "1";
  const question = "2";
