--- expected
+++ actual
@@ -7,6 +7,7 @@
 'use strict';
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
+  'a': function() { return _lib__WEBPACK_IMPORTED_MODULE_0_.answer; },
   'q': function() { return _lib__WEBPACK_IMPORTED_MODULE_0_.question; }
 });
 /* harmony import */var _lib__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./lib */"./lib.js");
@@ -24,6 +25,7 @@
 'use strict';
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
+  'answer': function() { return answer; },
   'question': function() { return question; }
 });
  const answer = "1";
