--- expected
+++ actual
@@ -29,8 +29,11 @@
 "./lib.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
 __webpack_require__.r(__webpack_exports__);
+__webpack_require__.d(__webpack_exports__, {
+  b: function() { return /* reexport safe */ _a_js__WEBPACK_IMPORTED_MODULE_0__.b; },
+  a: function() { return /* reexport safe */ _a_js__WEBPACK_IMPORTED_MODULE_0__.a; }
+});
 /* harmony import */var _a_js__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./a.js */"./a.js");
-__webpack_require__.es(_a_js__WEBPACK_IMPORTED_MODULE_0__, __webpack_exports__);
 
 }),
 
