--- expected
+++ actual
@@ -4,10 +4,11 @@
 ```js title=main.js
 (self['webpackChunkwebpack'] = self['webpackChunkwebpack'] || []).push([["main"], {
 "./zh_locale.js": (function (__unused_webpack_module, exports, __webpack_require__) {
+var __webpack_unused_export__;
 "use strict";
-Object.defineProperty(exports, "__esModule", ({
+__webpack_unused_export__ = ({
     value: true
-}));
+});
 exports["default"] = void 0;
 /* eslint-disable no-template-curly-in-string */ var _default = {};
 exports["default"] = _default;
@@ -28,9 +29,6 @@
 "./index.ts": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
 __webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  test: function() { return test; }
-});
 /* harmony import */var _antd_index__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./antd/index */"./antd/index.ts");
 
 _antd_index__WEBPACK_IMPORTED_MODULE_0__.locales.zh_CN;
