--- expected
+++ actual
@@ -7,7 +7,7 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  something: function() { return _lib__WEBPACK_IMPORTED_MODULE_0__["default"]; }
+  something: function() { return /* reexport safe */ _lib__WEBPACK_IMPORTED_MODULE_0__["default"]; }
 });
 /* harmony import */var _lib__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./lib */"./lib.js");
 /* harmony import */var _src_a__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/*! ./src/a */"./src/a.js");
