--- expected
+++ actual
@@ -7,7 +7,7 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  something: function() { return _lib__WEBPACK_IMPORTED_MODULE_0__["default"]; }
+  E: function() { return _lib__WEBPACK_IMPORTED_MODULE_0__.ZP; }
 });
 /* harmony import */var _lib__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./lib */"./lib.js");
 /* harmony import */var _src_a__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/*! ./src/a */"./src/a.js");
@@ -19,13 +19,13 @@
 __webpack_require__.r(__webpack_exports__);
 /* harmony import */var _app__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./app */"./app.js");
 
-(0, _app__WEBPACK_IMPORTED_MODULE_0__.something)();
+(0, _app__WEBPACK_IMPORTED_MODULE_0__.E)();
 }),
 "./lib.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  "default": function() { return __WEBPACK_DEFAULT_EXPORT__; }
+  ZP: function() { return __WEBPACK_DEFAULT_EXPORT__; }
 });
  const secret = "888";
  const result = 20000;
