--- expected
+++ actual
@@ -37,15 +37,6 @@
 }
 
 }),
-"../../../../../node_modules/@swc/helpers/esm/_ts_generator.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
-"use strict";
-__webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  _: function() { return tslib__WEBPACK_IMPORTED_MODULE_0__.__generator; }
-});
-/* harmony import */var tslib__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! tslib */"../../../../../node_modules/tslib/tslib.es6.js");
-
-}),
 "../../../../../node_modules/tslib/tslib.es6.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
 __webpack_require__.r(__webpack_exports__);
@@ -482,11 +473,8 @@
 "./index.ts": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
 __webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  test: function() { return test; }
-});
 /* harmony import */var _swc_helpers_async_to_generator__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! @swc/helpers/_/_async_to_generator */"../../../../../node_modules/@swc/helpers/esm/_async_to_generator.js");
-/* harmony import */var _swc_helpers_ts_generator__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/*! @swc/helpers/_/_ts_generator */"../../../../../node_modules/@swc/helpers/esm/_ts_generator.js");
+/* harmony import */var _swc_helpers_ts_generator__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/*! @swc/helpers/_/_ts_generator */"../../../../../node_modules/tslib/tslib.es6.js");
 
 
  function test() {
@@ -494,7 +482,7 @@
 }
 function _test() {
     _test = (0, _swc_helpers_async_to_generator__WEBPACK_IMPORTED_MODULE_0__._)(function() {
-        return (0, _swc_helpers_ts_generator__WEBPACK_IMPORTED_MODULE_1__._)(this, function(_state) {
+        return (0, _swc_helpers_ts_generator__WEBPACK_IMPORTED_MODULE_1__.__generator)(this, function(_state) {
             return [
                 2
             ];
