--- expected
+++ actual
@@ -41,17 +41,14 @@
 'use strict';
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  '_': function() { return tslib__WEBPACK_IMPORTED_MODULE_0_.__generator; }
+  '_': function() { return tslib__WEBPACK_IMPORTED_MODULE_0_.__generator; },
+  '_ts_generator': function() { return tslib__WEBPACK_IMPORTED_MODULE_0_.__generator; }
 });
-/* harmony import */var tslib__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* tslib */"../../../../../node_modules/tslib/tslib.es6.js");
 
 },
 "../../../../../node_modules/tslib/tslib.es6.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  '__generator': function() { return __generator; }
-});
 /******************************************************************************
 Copyright (c) Microsoft Corporation.
 
@@ -482,9 +479,6 @@
 "./index.ts": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  'test': function() { return test; }
-});
 /* harmony import */var _swc_helpers_async_to_generator__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* @swc/helpers/_/_async_to_generator */"../../../../../node_modules/@swc/helpers/esm/_async_to_generator.js");
 /* harmony import */var _swc_helpers_ts_generator__WEBPACK_IMPORTED_MODULE_1_ = __webpack_require__(/* @swc/helpers/_/_ts_generator */"../../../../../node_modules/@swc/helpers/esm/_ts_generator.js");
 
