--- expected
+++ actual
@@ -7,6 +7,7 @@
 'use strict';
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
+  '_async_to_generator': function() { return _async_to_generator; },
   '_': function() { return _async_to_generator; }
 });
 function asyncGeneratorStep(gen, resolve, reject, _next, _throw, key, arg) {
@@ -41,7 +42,8 @@
 'use strict';
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  '_': function() { return tslib__WEBPACK_IMPORTED_MODULE_0_.__generator; }
+  '_': function() { return tslib__WEBPACK_IMPORTED_MODULE_0_.__generator; },
+  '_ts_generator': function() { return tslib__WEBPACK_IMPORTED_MODULE_0_.__generator; }
 });
 /* harmony import */var tslib__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* tslib */"../../../../../node_modules/tslib/tslib.es6.js");
 
@@ -50,7 +52,35 @@
 'use strict';
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  '__generator': function() { return __generator; }
+  '__extends': function() { return __extends; },
+  '__assign': function() { return __assign; },
+  '__rest': function() { return __rest; },
+  '__decorate': function() { return __decorate; },
+  '__param': function() { return __param; },
+  '__esDecorate': function() { return __esDecorate; },
+  '__runInitializers': function() { return __runInitializers; },
+  '__propKey': function() { return __propKey; },
+  '__setFunctionName': function() { return __setFunctionName; },
+  '__metadata': function() { return __metadata; },
+  '__awaiter': function() { return __awaiter; },
+  '__generator': function() { return __generator; },
+  '__createBinding': function() { return __createBinding; },
+  '__exportStar': function() { return __exportStar; },
+  '__values': function() { return __values; },
+  '__read': function() { return __read; },
+  '__spread': function() { return __spread; },
+  '__spreadArrays': function() { return __spreadArrays; },
+  '__spreadArray': function() { return __spreadArray; },
+  '__await': function() { return __await; },
+  '__asyncGenerator': function() { return __asyncGenerator; },
+  '__asyncDelegator': function() { return __asyncDelegator; },
+  '__asyncValues': function() { return __asyncValues; },
+  '__makeTemplateObject': function() { return __makeTemplateObject; },
+  '__importStar': function() { return __importStar; },
+  '__importDefault': function() { return __importDefault; },
+  '__classPrivateFieldGet': function() { return __classPrivateFieldGet; },
+  '__classPrivateFieldSet': function() { return __classPrivateFieldSet; },
+  '__classPrivateFieldIn': function() { return __classPrivateFieldIn; }
 });
 /******************************************************************************
 Copyright (c) Microsoft Corporation.
