--- expected
+++ actual
@@ -19,6 +19,9 @@
 "./foo.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
+__webpack_require__.d(__webpack_exports__, {
+  'batch': function() { return batch; }
+});
 /* harmony import */var _app__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./app */"./app.js");
 __webpack_require__.es(_app__WEBPACK_IMPORTED_MODULE_0_, __webpack_exports__);
 
