--- expected
+++ actual
@@ -3,41 +3,20 @@
 ---
 ```js title=main.js
 (self['webpackChunkwebpack'] = self['webpackChunkwebpack'] || []).push([["main"], {
-"./app.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
-"use strict";
-__webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  Provider: function() { return _lib__WEBPACK_IMPORTED_MODULE_0__["default"]; },
-  useSelector: function() { return _selector_js__WEBPACK_IMPORTED_MODULE_1__["default"]; }
-});
-/* harmony import */var _lib__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./lib */"./lib.js");
-/* harmony import */var _selector_js__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/*! ./selector.js */"./selector.js");
-
-
-
-}),
-"./foo.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
-"use strict";
-__webpack_require__.r(__webpack_exports__);
-/* harmony import */var _app__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./app */"./app.js");
-__webpack_require__.es(_app__WEBPACK_IMPORTED_MODULE_0__, __webpack_exports__);
-
-function batch() {}
-
-}),
 "./index.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
 __webpack_require__.r(__webpack_exports__);
-/* harmony import */var _foo__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./foo */"./foo.js");
+/* harmony import */var _foo__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./foo */"./lib.js");
+/* harmony import */var _foo__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/*! ./foo */"./selector.js");
 
-_foo__WEBPACK_IMPORTED_MODULE_0__.Provider;
-_foo__WEBPACK_IMPORTED_MODULE_0__.useSelector;
+_foo__WEBPACK_IMPORTED_MODULE_0__.Z;
+_foo__WEBPACK_IMPORTED_MODULE_1__.Z;
 }),
 "./lib.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  "default": function() { return __WEBPACK_DEFAULT_EXPORT__; }
+  Z: function() { return __WEBPACK_DEFAULT_EXPORT__; }
 });
 function Provider() {}
 var __WEBPACK_DEFAULT_EXPORT__ = Provider;
@@ -46,7 +25,7 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  "default": function() { return useSelector; }
+  Z: function() { return useSelector; }
 });
 function useSelector() {
     return "";
