--- expected
+++ actual
@@ -36,18 +36,12 @@
 "./lib.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  'default': function() { return __WEBPACK_DEFAULT_EXPORT__; }
-});
 function Provider() {}
 var __WEBPACK_DEFAULT_EXPORT__ = Provider;
 },
 "./selector.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  'default': function() { return useSelector; }
-});
 function useSelector() {
     return "";
 }
