--- expected
+++ actual
@@ -7,7 +7,7 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  "default": function() { return __WEBPACK_DEFAULT_EXPORT__; }
+  Z: function() { return __WEBPACK_DEFAULT_EXPORT__; }
 });
 var Foo = function() {
     console.log("side effect");
@@ -25,7 +25,7 @@
 __webpack_require__.r(__webpack_exports__);
 /* harmony import */var _foo__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./foo */"./foo.js");
 
-var foo = new _foo__WEBPACK_IMPORTED_MODULE_0__["default"]();
+var foo = new _foo__WEBPACK_IMPORTED_MODULE_0__.Z();
 }),
 
 },function(__webpack_require__) {
