--- expected
+++ actual
@@ -16,7 +16,8 @@
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
   'xxx': function() { return _test_js__WEBPACK_IMPORTED_MODULE_0_; },
-  'square': function() { return square; }
+  'square': function() { return square; },
+  'cube': function() { return cube; }
 });
 /* harmony import */var _test_js__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./test.js */"./test.js");
 // maths.js
@@ -37,7 +38,8 @@
 'use strict';
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  'test': function() { return test; }
+  'test': function() { return test; },
+  'ccc': function() { return ccc; }
 });
  function test() {}
  function ccc() {}
