--- expected
+++ actual
@@ -16,7 +16,7 @@
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
   square: function() { return square; },
-  xxx: function() { return _test_js__WEBPACK_IMPORTED_MODULE_0__; }
+  xxx: function() { return /* reexport module object */ _test_js__WEBPACK_IMPORTED_MODULE_0__; }
 });
 /* harmony import */var _test_js__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./test.js */"./test.js");
 // maths.js
