--- expected
+++ actual
@@ -15,8 +15,7 @@
 'use strict';
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  'xxx': function() { return _test_js__WEBPACK_IMPORTED_MODULE_0_; },
-  'square': function() { return square; }
+  'xxx': function() { return _test_js__WEBPACK_IMPORTED_MODULE_0_; }
 });
 /* harmony import */var _test_js__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./test.js */"./test.js");
 // maths.js
@@ -36,9 +35,6 @@
 "./test.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  'test': function() { return test; }
-});
  function test() {}
  function ccc() {}
 },
