--- expected
+++ actual
@@ -14,6 +14,9 @@
 "./app.js": function (module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
+__webpack_require__.d(__webpack_exports__, {
+  'render': function() { return render; }
+});
 /* harmony import */var _lib__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./lib */"./lib.js");
 
 setTimeout(()=>{
@@ -22,7 +25,7 @@
  function render() {
     function test() {
         const container = document.getElementById("root");
-        container.innerHTML = `adddd333:${/* "./lib" unused */null}:${_lib__WEBPACK_IMPORTED_MODULE_0_.myanswer}`;
+        container.innerHTML = `adddd333:${_lib__WEBPACK_IMPORTED_MODULE_0_.secret}:${_lib__WEBPACK_IMPORTED_MODULE_0_.myanswer}`;
     }
 }
 if (module.hot?.accept) module.hot.accept((module1)=>{
@@ -39,7 +42,9 @@
 'use strict';
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  'myanswer': function() { return myanswer; }
+  'secret': function() { return secret; },
+  'myanswer': function() { return myanswer; },
+  'result': function() { return result; }
 });
 /* harmony import */var _answer__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./answer */"./answer.js");
 
