--- expected
+++ actual
@@ -22,7 +22,7 @@
  function render() {
     function test() {
         const container = document.getElementById("root");
-        container.innerHTML = `adddd333:${/* "./lib" unused */null}:${_lib__WEBPACK_IMPORTED_MODULE_0_.myanswer}`;
+        container.innerHTML = `adddd333:${_lib__WEBPACK_IMPORTED_MODULE_0_.secret}:${_lib__WEBPACK_IMPORTED_MODULE_0_.myanswer}`;
     }
 }
 if (module.hot?.accept) module.hot.accept((module1)=>{
@@ -39,6 +39,7 @@
 'use strict';
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
+  'secret': function() { return secret; },
   'myanswer': function() { return myanswer; }
 });
 /* harmony import */var _answer__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./answer */"./answer.js");
