--- expected
+++ actual
@@ -22,7 +22,7 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  answer: function() { return _answer__WEBPACK_IMPORTED_MODULE_0__.answer; }
+  answer: function() { return /* reexport safe */ _answer__WEBPACK_IMPORTED_MODULE_0__.answer; }
 });
 /* harmony import */var _answer__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./answer */"./answer.js");
 
