--- expected
+++ actual
@@ -21,8 +21,10 @@
 "./lib.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
 __webpack_require__.r(__webpack_exports__);
+__webpack_require__.d(__webpack_exports__, {
+  answer: function() { return /* reexport safe */ _answer__WEBPACK_IMPORTED_MODULE_0__.answer; }
+});
 /* harmony import */var _answer__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./answer */"./answer.js");
-__webpack_require__.es(_answer__WEBPACK_IMPORTED_MODULE_0__, __webpack_exports__);
 
 }),
 
