--- expected
+++ actual
@@ -38,7 +38,7 @@
     ]);
     return ConsoleExporterWeb;
 }();
-/* unused harmony default export */ var __WEBPACK_DEFAULT_EXPORT__ = (ConsoleExporterWeb);
+/* unused harmony default export */ var __WEBPACK_DEFAULT_EXPORT__ = ((/* unused pure expression or super */ null && (ConsoleExporterWeb)));
 }),
 "../../../../../node_modules/@swc/helpers/esm/_create_class.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
