--- expected
+++ actual
@@ -7,11 +7,11 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  "default": function() { return __WEBPACK_DEFAULT_EXPORT__; }
+  Z: function() { return __WEBPACK_DEFAULT_EXPORT__; }
 });
-/* harmony import */var _containers__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./containers */"./src/containers/index.js");
+/* harmony import */var _containers__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./containers */"./src/containers/containers.js");
 
-const { PlatformProvider } = _containers__WEBPACK_IMPORTED_MODULE_0__.containers;
+const { PlatformProvider } = _containers__WEBPACK_IMPORTED_MODULE_0__;
 const Index = ()=>{
     console.log("PlatformProvider", PlatformProvider);
     return 'something';
@@ -23,16 +23,6 @@
 __webpack_require__.r(__webpack_exports__);
 /* harmony import */var _platform_container__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./platform-container */"./src/containers/platform-container/index.js");
 __webpack_require__.es(_platform_container__WEBPACK_IMPORTED_MODULE_0__, __webpack_exports__);
-
-
-}),
-"./src/containers/index.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
-"use strict";
-__webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  containers: function() { return _containers__WEBPACK_IMPORTED_MODULE_0__; }
-});
-/* harmony import */var _containers__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./containers */"./src/containers/containers.js");
 
 
 }),
@@ -40,8 +30,8 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  usePlatform: function() { return usePlatform; },
-  PlatformProvider: function() { return PlatformProvider; }
+  x: function() { return usePlatform; },
+  P: function() { return PlatformProvider; }
 });
  const usePlatform = 3;
  const PlatformProvider = 1000;
@@ -51,7 +41,7 @@
 __webpack_require__.r(__webpack_exports__);
 /* harmony import */var _App__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./App */"./src/App.js");
 
-(0, _App__WEBPACK_IMPORTED_MODULE_0__["default"])();
+(0, _App__WEBPACK_IMPORTED_MODULE_0__.Z)();
 }),
 
 },function(__webpack_require__) {
