--- expected
+++ actual
@@ -9,7 +9,6 @@
 __webpack_require__.d(__webpack_exports__, {
   'formatNumber': function() { return _plugin_formatNumber__WEBPACK_IMPORTED_MODULE_0_["default"]; }
 });
-/* harmony import */var _plugin_formatNumber__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./plugin/formatNumber */"./src/plugin/formatNumber.js");
 
 
 },
@@ -23,9 +22,6 @@
 "./src/plugin/formatNumber.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  'default': function() { return formatNumber_default; }
-});
 function formatNumber(config) {}
 const plugin = (cls)=>{
     cls.prototype.formatNumber = formatNumber;
