--- expected
+++ actual
@@ -23,9 +23,6 @@
 "./src/plugin/formatNumber.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  'default': function() { return formatNumber_default; }
-});
 function formatNumber(config) {}
 const plugin = (cls)=>{
     cls.prototype.formatNumber = formatNumber;
