--- expected
+++ actual
@@ -7,17 +7,17 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  aaa: function() { return _app__WEBPACK_IMPORTED_MODULE_1__; },
-  routes: function() { return routes; }
+  P: function() { return _app__WEBPACK_IMPORTED_MODULE_0__; },
+  _: function() { return routes; }
 });
-/* harmony import */var _answer__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./answer */"./answer.js");
-/* harmony import */var _answer__WEBPACK_IMPORTED_MODULE_0___default = /*#__PURE__*/__webpack_require__.n(_answer__WEBPACK_IMPORTED_MODULE_0__);
-/* harmony import */var _app__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/*! ./app */"./app.js");
-/* harmony import */var _app__WEBPACK_IMPORTED_MODULE_1___default = /*#__PURE__*/__webpack_require__.n(_app__WEBPACK_IMPORTED_MODULE_1__);
+/* harmony import */var _answer__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/*! ./answer */"./answer.js");
+/* harmony import */var _answer__WEBPACK_IMPORTED_MODULE_1___default = /*#__PURE__*/__webpack_require__.n(_answer__WEBPACK_IMPORTED_MODULE_1__);
+/* harmony import */var _app__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./app */"./app.js");
+/* harmony import */var _app__WEBPACK_IMPORTED_MODULE_0___default = /*#__PURE__*/__webpack_require__.n(_app__WEBPACK_IMPORTED_MODULE_0__);
 
 
  const routes = {
-    answer: _answer__WEBPACK_IMPORTED_MODULE_0__.something
+    answer: _answer__WEBPACK_IMPORTED_MODULE_1__.something
 };
 }),
 "./answer.js": (function (__unused_webpack_module, exports, __webpack_require__) {
@@ -29,8 +29,8 @@
 __webpack_require__.r(__webpack_exports__);
 /* harmony import */var _a_js__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./a.js */"./a.js");
 
-_a_js__WEBPACK_IMPORTED_MODULE_0__.routes;
-_a_js__WEBPACK_IMPORTED_MODULE_0__.aaa.result;
+_a_js__WEBPACK_IMPORTED_MODULE_0__._;
+_a_js__WEBPACK_IMPORTED_MODULE_0__.P.result;
 }),
 
 },function(__webpack_require__) {
