--- expected
+++ actual
@@ -15,15 +15,6 @@
     return _dep_a__WEBPACK_IMPORTED_MODULE_0_.x;
 }
 },
-"./b.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
-'use strict';
-__webpack_require__.r(__webpack_exports__);
-/* harmony import */var _dep_b__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./dep?b */"./dep.js?b");
-
-function abc() {
-    return /* "./dep?b" unused */null;
-}
-},
 "./c.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
@@ -56,7 +47,7 @@
   'default': function() { return __WEBPACK_DEFAULT_EXPORT__; }
 });
  const x = "x";
-var __WEBPACK_DEFAULT_EXPORT__ = true;
+var __WEBPACK_DEFAULT_EXPORT__ = false;
 },
 "./dep.js?b": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
@@ -75,7 +66,7 @@
   'default': function() { return __WEBPACK_DEFAULT_EXPORT__; }
 });
  const x = "x";
-var __WEBPACK_DEFAULT_EXPORT__ = true;
+var __WEBPACK_DEFAULT_EXPORT__ = false;
 },
 "./dep.js?d": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
@@ -85,7 +76,7 @@
   'default': function() { return __WEBPACK_DEFAULT_EXPORT__; }
 });
  const x = "x";
-var __WEBPACK_DEFAULT_EXPORT__ = true;
+var __WEBPACK_DEFAULT_EXPORT__ = false;
 },
 "./dep.js?e": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
@@ -104,19 +95,8 @@
   'default': function() { return __WEBPACK_DEFAULT_EXPORT__; }
 });
  const x = "x";
-var __WEBPACK_DEFAULT_EXPORT__ = true;
+var __WEBPACK_DEFAULT_EXPORT__ = false;
 },
-"./e.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
-'use strict';
-__webpack_require__.r(__webpack_exports__);
-/* harmony import */var _dep_e__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./dep?e */"./dep.js?e");
-
-class def {
-    method() {
-        return /* "./dep?e" unused */null;
-    }
-}
-},
 "./f.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
@@ -132,12 +112,10 @@
 "./index.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
+/* harmony import */var _c__WEBPACK_IMPORTED_MODULE_2_ = __webpack_require__(/* ./c */"./c.js");
+/* harmony import */var _f__WEBPACK_IMPORTED_MODULE_5_ = __webpack_require__(/* ./f */"./f.js");
 /* harmony import */var _a__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./a */"./a.js");
-/* harmony import */var _b__WEBPACK_IMPORTED_MODULE_1_ = __webpack_require__(/* ./b */"./b.js");
-/* harmony import */var _c__WEBPACK_IMPORTED_MODULE_2_ = __webpack_require__(/* ./c */"./c.js");
 /* harmony import */var _d__WEBPACK_IMPORTED_MODULE_3_ = __webpack_require__(/* ./d */"./d.js");
-/* harmony import */var _e__WEBPACK_IMPORTED_MODULE_4_ = __webpack_require__(/* ./e */"./e.js");
-/* harmony import */var _f__WEBPACK_IMPORTED_MODULE_5_ = __webpack_require__(/* ./f */"./f.js");
 /* harmony import */var _dep_a__WEBPACK_IMPORTED_MODULE_6_ = __webpack_require__(/* ./dep?a */"./dep.js?a");
 /* harmony import */var _dep_b__WEBPACK_IMPORTED_MODULE_7_ = __webpack_require__(/* ./dep?b */"./dep.js?b");
 /* harmony import */var _dep_c__WEBPACK_IMPORTED_MODULE_8_ = __webpack_require__(/* ./dep?c */"./dep.js?c");
