--- expected
+++ actual
@@ -21,7 +21,7 @@
 /* harmony import */var _dep_b__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./dep?b */"./dep.js?b");
 
 function abc() {
-    return /* "./dep?b" unused */null;
+    return x;
 }
 }),
 "./c.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
@@ -56,7 +56,7 @@
   "default": function() { return __WEBPACK_DEFAULT_EXPORT__; }
 });
  const x = "x";
-var __WEBPACK_DEFAULT_EXPORT__ = true;
+var __WEBPACK_DEFAULT_EXPORT__ = false;
 }),
 "./dep.js?b": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
@@ -75,7 +75,7 @@
   "default": function() { return __WEBPACK_DEFAULT_EXPORT__; }
 });
  const x = "x";
-var __WEBPACK_DEFAULT_EXPORT__ = true;
+var __WEBPACK_DEFAULT_EXPORT__ = false;
 }),
 "./dep.js?d": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
@@ -85,7 +85,7 @@
   "default": function() { return __WEBPACK_DEFAULT_EXPORT__; }
 });
  const x = "x";
-var __WEBPACK_DEFAULT_EXPORT__ = true;
+var __WEBPACK_DEFAULT_EXPORT__ = false;
 }),
 "./dep.js?e": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
@@ -104,7 +104,7 @@
   "default": function() { return __WEBPACK_DEFAULT_EXPORT__; }
 });
  const x = "x";
-var __WEBPACK_DEFAULT_EXPORT__ = true;
+var __WEBPACK_DEFAULT_EXPORT__ = false;
 }),
 "./e.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
@@ -113,7 +113,7 @@
 
 class def {
     method() {
-        return /* "./dep?e" unused */null;
+        return x;
     }
 }
 }),
