--- expected
+++ actual
@@ -12,6 +12,16 @@
  function a() {}
  function dddd() {}
 },
+"./package/autogen/b.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
+'use strict';
+__webpack_require__.r(__webpack_exports__);
+ function b() {}
+},
+"./package/autogen/c.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
+'use strict';
+__webpack_require__.r(__webpack_exports__);
+ function c() {}
+},
 "./package/autogen/index.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
@@ -19,6 +29,8 @@
   'a': function() { return _a__WEBPACK_IMPORTED_MODULE_0_; }
 });
 /* harmony import */var _a__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./a */"./package/autogen/a.js");
+/* harmony import */var _b__WEBPACK_IMPORTED_MODULE_1_ = __webpack_require__(/* ./b */"./package/autogen/b.js");
+/* harmony import */var _c__WEBPACK_IMPORTED_MODULE_2_ = __webpack_require__(/* ./c */"./package/autogen/c.js");
 
 
 
