--- expected
+++ actual
@@ -3,9 +3,31 @@
 ---
 ```js title=main.js
 (self['webpackChunkwebpack'] = self['webpackChunkwebpack'] || []).push([["main"], {
+"./package/autogen/a.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
+"use strict";
+__webpack_require__.r(__webpack_exports__);
+__webpack_require__.d(__webpack_exports__, {
+  a: function() { return a; }
+});
+ function a() {}
+ function dddd() {}
+}),
+"./package/autogen/index.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
+"use strict";
+__webpack_require__.r(__webpack_exports__);
+__webpack_require__.d(__webpack_exports__, {
+  a: function() { return _a__WEBPACK_IMPORTED_MODULE_0__; }
+});
+/* harmony import */var _a__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/* ./a */"./package/autogen/a.js");
+
+
+
+}),
 "./package/src/index.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
 __webpack_require__.r(__webpack_exports__);
+/* harmony import */var _autogen_index__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/* ../autogen/index */"./package/autogen/index.js");
+__webpack_require__.es(_autogen_index__WEBPACK_IMPORTED_MODULE_0__, __webpack_exports__);
 
 }),
 "./src/index.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
