--- expected
+++ actual
@@ -19,9 +19,8 @@
   a: function() { return _a__WEBPACK_IMPORTED_MODULE_0_; }
 });
 /* harmony import */var _a__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./a */"./package/autogen/a.js");
-
-
-
+ // export * as b  from './b'
+ // export * as c from './c'
 },
 "./package/src/index.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
