--- expected
+++ actual
@@ -15,7 +15,7 @@
     _lib__WEBPACK_IMPORTED_MODULE_0_.result;
 };
  var app2 = ()=>{
-    /* "./lib" unused */null;
+    _lib__WEBPACK_IMPORTED_MODULE_0_.secret;
 };
  var app4 = (0, _lib__WEBPACK_IMPORTED_MODULE_0_.something)('app4'), app5 = 10000;
  var app3 = (0, _lib__WEBPACK_IMPORTED_MODULE_0_.something)('app3');
@@ -30,10 +30,6 @@
 "./lib.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  'result': function() { return result; },
-  'something': function() { return something; }
-});
  const secret = "888";
  const result = 20000;
  const something = function() {};
