--- expected
+++ actual
@@ -6,11 +6,20 @@
 "./a.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  'default': function() { return __WEBPACK_DEFAULT_EXPORT__; }
-});
 var __WEBPACK_DEFAULT_EXPORT__ = 300;
 },
+"./app.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
+'use strict';
+__webpack_require__.r(__webpack_exports__);
+ function app() {}
+ function test() {}
+},
+"./b.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
+'use strict';
+__webpack_require__.r(__webpack_exports__);
+__webpack_require__(/* ./app.js */"./app.js");
+var __WEBPACK_DEFAULT_EXPORT__ = 200;
+},
 "./index.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
@@ -22,9 +31,9 @@
 'use strict';
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  'a': function() { return _a_js__WEBPACK_IMPORTED_MODULE_0_["default"]; }
+  'a': function() { return _a_js__WEBPACK_IMPORTED_MODULE_0_["default"]; },
+  'b': function() { return _b_js__WEBPACK_IMPORTED_MODULE_1_["default"]; }
 });
-/* harmony import */var _a_js__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./a.js */"./a.js");
 
 
 },
