--- expected
+++ actual
@@ -6,9 +6,6 @@
 "./a.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  'default': function() { return __WEBPACK_DEFAULT_EXPORT__; }
-});
 var __WEBPACK_DEFAULT_EXPORT__ = 300;
 },
 "./index.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
