--- expected
+++ actual
@@ -3,12 +3,11 @@
 ---
 ```js title=main.js
 (self['webpackChunkwebpack'] = self['webpackChunkwebpack'] || []).push([["main"], {
-"./index.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
+"./index.js": (function () {
 "use strict";
-__webpack_require__.r(__webpack_exports__);
 
 function test() {
-    /* "./lib.js" unused */null;
+    a;
 }
 }),
 
