--- expected
+++ actual
@@ -8,23 +8,24 @@
 __webpack_require__.r(__webpack_exports__);
 /* harmony import */var _something__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./something */"./something/index.js");
 __webpack_require__.es(_something__WEBPACK_IMPORTED_MODULE_0_, __webpack_exports__);
+/* harmony import */var _c_js__WEBPACK_IMPORTED_MODULE_1_ = __webpack_require__(/* ./c.js */"./c.js");
+__webpack_require__.es(_c_js__WEBPACK_IMPORTED_MODULE_1_, __webpack_exports__);
 
 
 },
+"./c.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
+'use strict';
+__webpack_require__.r(__webpack_exports__);
+ function cccc() {}
+},
 "./colors/a.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  'red': function() { return red; }
-});
  const red = 'red';
 },
 "./colors/b.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  'blue': function() { return blue; }
-});
  const blue = 'blue';
 },
 "./colors/c.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
@@ -50,9 +51,6 @@
 "./colors/result.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  'result': function() { return result; }
-});
  const result = 'ssss';
 },
 "./export.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
@@ -73,9 +71,6 @@
 "./something/Something.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  'Something': function() { return Something; }
-});
  class Something {
 }
 },
