--- expected
+++ actual
@@ -7,11 +7,12 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  bar: function() { return _foo__WEBPACK_IMPORTED_MODULE_0__; }
+  b: function() { return b; },
+  bar: function() { return /* reexport module object */ _foo__WEBPACK_IMPORTED_MODULE_0__; },
+  c: function() { return /* reexport safe */ _result__WEBPACK_IMPORTED_MODULE_1__.c; }
 });
 /* harmony import */var _foo__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./foo */"./foo.js");
 /* harmony import */var _result__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/*! ./result */"./result.js");
-__webpack_require__.es(_result__WEBPACK_IMPORTED_MODULE_1__, __webpack_exports__);
  function b() {}
 
 
@@ -20,12 +21,14 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  a: function() { return a; }
+  a: function() { return a; },
+  b: function() { return /* reexport safe */ _bar__WEBPACK_IMPORTED_MODULE_0__.b; },
+  bar: function() { return /* reexport safe */ _bar__WEBPACK_IMPORTED_MODULE_0__.bar; },
+  c: function() { return /* reexport safe */ _bar__WEBPACK_IMPORTED_MODULE_0__.c; },
+  foo: function() { return foo; }
 });
 /* harmony import */var _bar__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./bar */"./bar.js");
-__webpack_require__.es(_bar__WEBPACK_IMPORTED_MODULE_0__, __webpack_exports__);
 /* harmony import */var _result__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/*! ./result */"./result.js");
-__webpack_require__.es(_result__WEBPACK_IMPORTED_MODULE_1__, __webpack_exports__);
  const a = 3;
  const foo = 3;
 
@@ -46,9 +49,7 @@
   c: function() { return c; }
 });
 /* harmony import */var _foo__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./foo */"./foo.js");
-__webpack_require__.es(_foo__WEBPACK_IMPORTED_MODULE_0__, __webpack_exports__);
 /* harmony import */var _bar__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/*! ./bar */"./bar.js");
-__webpack_require__.es(_bar__WEBPACK_IMPORTED_MODULE_1__, __webpack_exports__);
  const c = 103330;
 
 
