--- expected
+++ actual
@@ -6,7 +6,13 @@
 "./bar.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
 __webpack_require__.r(__webpack_exports__);
+__webpack_require__.d(__webpack_exports__, {
+  bar: function() { return _foo__WEBPACK_IMPORTED_MODULE_0__; },
+  b: function() { return b; }
+});
 /* harmony import */var _foo__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/* ./foo */"./foo.js");
+/* harmony import */var _result__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/* ./result */"./result.js");
+__webpack_require__.es(_result__WEBPACK_IMPORTED_MODULE_1__, __webpack_exports__);
  function b() {}
 
 
@@ -14,6 +20,12 @@
 "./foo.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
 __webpack_require__.r(__webpack_exports__);
+__webpack_require__.d(__webpack_exports__, {
+  a: function() { return a; },
+  foo: function() { return foo; }
+});
+/* harmony import */var _bar__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/* ./bar */"./bar.js");
+__webpack_require__.es(_bar__WEBPACK_IMPORTED_MODULE_0__, __webpack_exports__);
  const a = 3;
  const foo = 3;
 
@@ -30,6 +42,9 @@
 "./result.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
 __webpack_require__.r(__webpack_exports__);
+__webpack_require__.d(__webpack_exports__, {
+  c: function() { return c; }
+});
  const c = 103330;
 
 
