--- expected
+++ actual
@@ -7,7 +7,7 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  something: function() { return something; }
+  E: function() { return something; }
 });
  function something() {}
 }),
@@ -15,13 +15,13 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  "default": function() { return a; }
+  Z: function() { return a; }
 });
 /* harmony import */var _Something__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./Something */"./Something.js");
 
 
 var a = function test() {
-    _Something__WEBPACK_IMPORTED_MODULE_0__.something;
+    _Something__WEBPACK_IMPORTED_MODULE_0__.E;
 };
 
 }),
@@ -30,7 +30,7 @@
 __webpack_require__.r(__webpack_exports__);
 /* harmony import */var _export__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./export */"./export.js");
 
-_export__WEBPACK_IMPORTED_MODULE_0__["default"];
+_export__WEBPACK_IMPORTED_MODULE_0__.Z;
 }),
 
 },function(__webpack_require__) {
