--- expected
+++ actual
@@ -10,10 +10,6 @@
 "./source/index.js": function (module, exports, __webpack_require__) {
 var _class_call_check = __webpack_require__(/* @swc/helpers/_/_class_call_check */"../../../../../node_modules/@swc/helpers/esm/_class_call_check.js");
 var _create_class = __webpack_require__(/* @swc/helpers/_/_create_class */"../../../../../node_modules/@swc/helpers/esm/_create_class.js");
-var test = function test() {
-    var res = new Response();
-    return res;
-};
 var Response = function() {
     "use strict";
     function Response(mode) {
@@ -33,6 +29,10 @@
     ]);
     return Response;
 }();
+function test() {
+    var res = new Response();
+    return res;
+}
 var result = test();
 module.exports = result;
 },
