--- expected
+++ actual
@@ -12,23 +12,12 @@
 class Something {
 }
 }),
-"./export.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
-"use strict";
-__webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  Sider: function() { return _Something__WEBPACK_IMPORTED_MODULE_0__["default"]; }
-});
-/* harmony import */var _Something__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./Something */"./Something.js");
-
-
-
-}),
 "./index.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
 __webpack_require__.r(__webpack_exports__);
-/* harmony import */var _export__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./export */"./export.js");
+/* harmony import */var _export__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./export */"./Something.js");
 
-(0, _export__WEBPACK_IMPORTED_MODULE_0__.Sider)();
+(0, _export__WEBPACK_IMPORTED_MODULE_0__["default"])();
 }),
 
 },function(__webpack_require__) {
