--- expected
+++ actual
@@ -3,6 +3,20 @@
 ---
 ```js title=main.js
 (self['webpackChunkwebpack'] = self['webpackChunkwebpack'] || []).push([["main"], {
+"../node_modules/pmodule/a.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
+"use strict";
+__webpack_require__.r(__webpack_exports__);
+__webpack_require__.d(__webpack_exports__, {
+  a: function() { return a; }
+});
+/* harmony import */var _tracker__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/* ./tracker */"../node_modules/pmodule/tracker.js");
+var a = "a";
+var b = "b";
+var c = "c";
+
+
+(0, _tracker__WEBPACK_IMPORTED_MODULE_0__.track)("a.js");
+}),
 "../node_modules/pmodule/b.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
 __webpack_require__.r(__webpack_exports__);
@@ -39,6 +53,8 @@
   z: function() { return _b__WEBPACK_IMPORTED_MODULE_1__.z; },
   "default": function() { return __WEBPACK_DEFAULT_EXPORT__; }
 });
+/* harmony import */var _a__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/* ./a */"../node_modules/pmodule/a.js");
+__webpack_require__.es(_a__WEBPACK_IMPORTED_MODULE_0__, __webpack_exports__);
 /* harmony import */var _b__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/* ./b */"../node_modules/pmodule/b.js");
 /* harmony import */var _tracker__WEBPACK_IMPORTED_MODULE_2__ = __webpack_require__(/* ./tracker */"../node_modules/pmodule/tracker.js");
 
