--- expected
+++ actual
@@ -6,9 +6,6 @@
 "../node_modules/pmodule/a.js": function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
-__webpack_require__.d(__webpack_exports__, {
-  'a': function() { return a; }
-});
 /* harmony import */var _tracker__WEBPACK_IMPORTED_MODULE_0_ = __webpack_require__(/* ./tracker */"../node_modules/pmodule/tracker.js");
 var a = "a";
 var b = "b";
@@ -50,6 +47,7 @@
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
   'x': function() { return _b__WEBPACK_IMPORTED_MODULE_1_.x; },
+  'y': function() { return _b__WEBPACK_IMPORTED_MODULE_1_.y; },
   'z': function() { return _b__WEBPACK_IMPORTED_MODULE_1_.z; },
   'default': function() { return __WEBPACK_DEFAULT_EXPORT__; }
 });
