--- expected
+++ actual
@@ -14,7 +14,7 @@
 "use strict";
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  "default": function() { return __WEBPACK_DEFAULT_EXPORT__; }
+  Z: function() { return __WEBPACK_DEFAULT_EXPORT__; }
 });
 /* harmony import */var _dead__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./dead */"./dead.js");
 
@@ -22,7 +22,7 @@
     return "foo";
 }
  function foodead() {
-    return "foo" + /* "./dead" unused */null();
+    return "foo" + dead();
 }
 }),
 "./index.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
@@ -30,7 +30,7 @@
 __webpack_require__.r(__webpack_exports__);
 /* harmony import */var _foo__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./foo */"./foo.js");
 
-assert.equal((0, _foo__WEBPACK_IMPORTED_MODULE_0__["default"])(), "foo");
+assert.equal((0, _foo__WEBPACK_IMPORTED_MODULE_0__.Z)(), "foo");
 }),
 
 },function(__webpack_require__) {
